# Q4
Tele call uses 4 centres around the globe to process customer order forms.They audit a certain % of the customer order forms . Any error in order form render it defective and has to be reworked before processing .The manager wants to check whether the defective % varies by centre .please analyze the data at 5% signiuficance level and help the manager draw appropriate inferences

In [4]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm
from scipy .stats import chi2_contingency

In [5]:
df=pd.read_csv('/content/Costomer+OrderForm.csv')
df.head()

Phillippines   Indonesia       Malta       India
0   Error Free  Error Free   Defective  Error Free
1   Error Free  Error Free  Error Free   Defective
2   Error Free   Defective   Defective  Error Free
3   Error Free  Error Free  Error Free  Error Free
4   Error Free  Error Free   Defective  Error Free

In [8]:
df.Phillippines.value_counts()

Error Free    271
Defective      29
Name: Phillippines, dtype: int64

In [9]:
df.Indonesia.value_counts()

Error Free    267
Defective      33
Name: Indonesia, dtype: int64

In [10]:
df.Malta.value_counts()


Error Free    269
Defective      31
Name: Malta, dtype: int64

In [11]:
df.India.value_counts()

Error Free    280
Defective      20
Name: India, dtype: int64

In [12]:
#make a contingency table
obs=np.array([[271,267,269,280],[29,33,31,20]])
obs

array([[271, 267, 269, 280],
       [ 29,  33,  31,  20]])

array([[271,267,269,280],[29,33,31,20]]) Assume Null hypotheis as Ho:Independence of categorical variable (customer orders forms defective % does not varies by centre) Thus, Alternative hypothesis as Ha Dependance of categorical variables (customers order forms defective % varies by centre)

In [13]:
#chi2 continegency independence test
chi2_contingency(obs)  # o/p is (chi2 stats value,p_value,df,expected observation )

Chi2ContingencyResult(statistic=3.858960685820355, pvalue=0.2771020991233135, dof=3, expected_freq=array([[271.75, 271.75, 271.75, 271.75],
       [ 28.25,  28.25,  28.25,  28.25]]))

In [14]:
# compare p_value with α = 0.05

Inference: As (p_value =0.2771)>(α = 0.05) ; Accept Null Hypothesis I.e independence of categorical varibles Thus,customer order forms defective % does not varies by centyre